src/digital_identity_service/index.js

```diff
@@ -1,16 +1,103 @@
+'use strict';
+
+const config = require('../../config');
 const Validation = require('../yoti_common/validation');
 
+const { RequestBuilder } = require('../request/request.builder');
+const { Payload } = require('../request/payload');
+const ShareSessionCreateResult = require('./share.session.create.result');
+
+const PolicyBuilder = require('./policy/policy.builder');
+const WantedAttributeBuilder = require('./policy/wanted.attribute.builder');
+const WantedAnchorBuilder = require('./policy/wanted.anchor.builder');
+const ConstraintsBuilder = require('./policy/constraints.builder');
+const SourceConstraintBuilder = require('./policy/source.constraint.builder');
+const ExtensionBuilder = require('./extension/extension.builder');
+const LocationConstraintExtensionBuilder = require('./extension/location.constraint.extension.builder');
+const TransactionalFlowExtensionBuilder = require('./extension/transactional.flow.extension.builder');
+const ThirdPartyAttributeExtensionBuilder = require('./extension/third.party.attribute.extension.builder');
+const ShareSessionBuilder = require('./share.session.builder');
+const ShareNotificationBuilder = require('./share.notification.builder');
+const ShareSession = require('./share.session');
+
+const DEFAULT_API_URL = config.yoti.digitalIdentityApi;
+
 class DigitalIdentityService {
-  constructor(sdkId, pem, { apiUrl }) {
+  constructor(sdkId, pem, { apiUrl = DEFAULT_API_URL } = {}) {
     Validation.isString(sdkId, 'sdkId');
     Validation.notNullOrEmpty(pem, 'pem');
 
     this.sdkId = sdkId;
     this.pem = pem;
     this.apiUrl = apiUrl;
   }
+
+  /**
+   *
+   * @param shareSessionConfig
```

Comment:
This `shareSessionConfig`, it seems that this is an instance of `share.session`, which is itself build using `share.session.builder`.
The name is a bit problematic, as it looks like that this is a full fledged session, but really, this is only the configuration to create one.
In IDV, it is referred as `specification`, we could follow the same or use `configuration` ?